docs/elasticsearch/index-templates.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: default
+title: Index Templates
+parent: Elasticsearch
+nav_order: 5
+---
+
+# Index template
+
+An index template formats all newly created indexes to a one like you may already have by automatically picking up on a naming schema.
+
+For example, if you have log indices being continuously indexed every day, you can define an index template to have all these log indices with the same mappings and settings.
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## Create template
+
+To create an index template, use a POST request:
+
+```json
+POST _template
+```
+
+This command creates a template named `daily_logs` and applies it to any new index that matches the regular expression `logs-2020-01-*` and also adds it to the `my_logs` alias:
```

Comment:
"new index that matches the regular expression" is a bit vague. "new index whose name matches the regular expression" might work better.